projects/stargazer/main.cpp

```diff
@@ -279,9 +270,7 @@ int main(int argc, char* argv[])
         return -1;
     }
 
-#ifndef NO_DAEMON
     const auto startFile = settings.GetConfDir() + START_FILE;
```

Comment:
It seems that `startFile` is only used in `creat`. It makes sense to move it closer to the point of use.